framework/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Yii Framework 2 Change Log
 - Enh #15135: Automatic completion for help in bash and zsh (Valkeru)
 - Enh #14662: Added support for custom `Content-Type` specification to `yii\web\JsonResponseFormatter` (Kolyunya)
 - Enh #14568: Refactored migration templates to use `safeUp()` and `safeDown()` methods (Kolyunya)
+- Enh: Added `yii\filters\auth\HttpHeaderAuth`
```

Comment:
Issue number and github nickname should be here.

Reply:
There is no issue number, should I create one?